lesson_07/homework_07.py

```diff
@@ -33,33 +36,67 @@ def multiplication_table(number):
 """
 
 
+def two_numbers_sum(num_1: int, num_2: int) -> int:
+    """Calculate the sum of two numbers"""
+    return num_1 + num_2
+
+
 # task 3
 """  Написати функцію, яка розрахує середнє арифметичне списку чисел.
 """
 
+
+def average_of_list_numbers(number_list: list) -> float:
+    """Calculate arithmetic mean of list of numbers"""
+    return sum(number_list) / len(number_list)
+
+
 # task 4
 """  Написати функцію, яка приймає рядок та повертає його у зворотному порядку.
 """
 
+
+def reverse_string(string: str) -> str:
+    """Reverse the given string"""
+    result_list = []
+    for i in reversed(range(len(string))):
+        result_list.append(string[i])
+    result_string = ''.join(result_list)
+    return result_string
```

Comment:
Чому б не спростити?
```suggestion
    return reversed(string)
```

Reply:
якщо просто reversed(string) то поверне <reversed object at 0x10998ffd0>

тоді вже так 
return ''.join(reversed(list(string)))

а якщо зовсім спростити, то так
return string[::-1]